config/management/operational/README.md

```diff
@@ -0,0 +1,40 @@
+# Libra Operational Tool
+
+The Libra Config Manager provides a tool for operators to perform maintenance functions
+on the Libra blockchain post-genesis.  The functionality of the tool is
+dictated by the organization of nodes within the system:
+
+* An association account that maintains the set of validator owners and validator
+  operators.
+* Validator owners (OW) that have accounts on the blockchain. These accounts contain
+  a validator configuration and specify a validator operator.
+* Validator operators (OP) that have accounts on the blockchain. These
+  accounts have the ability to manipulate validator configuration.
+
+## Requirements
+
+Each individual instance, OW or OP, should have access to a secure storage
+solution. Those leveraging Libra Secure Storage can directly use this tool,
+those that do not will need to provide their own tooling.
+
+## The Tools
+
+While this is compiled as a single binary it provides several different facilities:
+
+* A means for bootstrapping an identity mapped to a local configuration file.
+  The identities are used to interact with local and remote secure storages.
+* Retrieving and submitting validator operator, validator operator, and validator
```

Comment:
typo: "submitting validator operator, validator operator, and .."

Reply:
lol, also looks like this was straight copied from the previous README